### PR TITLE
more info on cookie_domain in dev portal auth sessions

### PR DESCRIPTION
**Pull request by tlblessing**
https://konghq.atlassian.net/browse/DOCS-1045

Direct review links:

https://deploy-preview-2022--kongdocs.netlify.app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions/#configuration-to-use-the-sessions-plugin-with-the-dev-portal

https://deploy-preview-2022--kongdocs.netlify.app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions/#domains

Link in first paragraph of Session plugin back to Dev Portal auth session:

https://deploy-preview-2022--kongdocs.netlify.app/hub/kong-inc/session/

### CODE REVIEW

File: app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md [hunk 1]
```diff
@@ -82,3 +87,24 @@ portal_session_conf = {
     "cookie_secure":false
 }
 ```
+
+### Domains
+
+There is a dependency that the dev portal `portal_api_url` and the dev
+portal api `portal_gui_host` need to share a domain or subdomain. The following
```
Comment (lena-larionova) [inline app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md:94]: Do you need to say "there is a dependency"? If you simplify to "the dev portal `portal_api_url` and the devportal api `portal_gui_host` need to share a domain or subdomain. " it gets the same message across.

File: app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md [hunk 2]
```diff
@@ -60,6 +62,8 @@ The Session configuration is secure by default, so the cookie uses the [Secure,
 
 ## Example Configurations
 
+### HTTPS and same domain
```
[the remark this answers is not in the document]
Comment (tlblessing) [reply app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md:65]: Inc. with added article "the"

HTTPS with the same domain for API and GUI

File: app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md (hunk 1 above)
Comment (tlblessing) [reply app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md:94]: No I was just using @satwant17's language. Inc. with some edits:

 The dev portal `portal_api_url` and the dev
portal api `portal_gui_host` must share a domain or subdomain.

Comment (tlblessing) [reply app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md]: Done in commit e6da64b

File: app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tlblessing) [reply app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md]: Done in commit e6da64b

File: app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md [hunk 3]
```diff
@@ -39,6 +40,7 @@ portal_session_conf = {
    the Plugin renews the session; 600 by default.
 * `"cookie_secure":<SET_DEPENDING_ON_PROTOCOL>`: `true` by default. See [Session Security](#session-security) for
     exceptions.
+* `"cookie_domain":<SET_DEPENDING_ON_DOMAIN>:` Optional. See [Session Security](#session-security) for exceptions.
 * `"cookie_samesite":"<SET_DEPENDING_ON_DOMAIN>"`: `"Strict"` by default. See [Session Security](#session-security) for
     exceptions.
```
[the remark this answers is not in the document]
Comment (tlblessing) [reply app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md:45]: It seems like customers have issues when configuring the plugin for use with the Dev Portal, so it doesn't seem like exact or unnecessary duplicate information to me. I added an xref from the Session plugin to the Dev Portal topic. Some prefer tables, and some prefer lists. I think a table is fine for the plugin and a list is fine here. In the interest of time/my workload, stet on a table.

File: app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md [hunk 4]
```diff
@@ -82,3 +87,24 @@ portal_session_conf = {
     "cookie_secure":false
 }
 ```
+
+### Domains
+
+The dev portal `portal_api_url` and the dev
+portal api `portal_gui_host` must share a domain or subdomain. The following
```
Comment (lena-larionova) [inline app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md:94]: Isn't this backwards? 

```
The dev portal `portal_api_url` and the dev portal api `portal_gui_host`
```

GUI host is for the dev portal, and the API URL is for the API.

Comment (tlblessing) [reply app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md:94]: Doh!

**Comment by tlblessing** [comment]
> looks good for cookie_domain now, this should definitely help most customers 👍

Thank you @satwant17 for bringing this to our attention!

File: app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md (hunk 4 above)
Comment (tlblessing) [reply app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md]: Fixed in commit a736660

**Pull request merged**